### PR TITLE
fix: import only necessary proto files from panacea-core

### PR DESCRIPTION
**Pull request by youngjoon-lee**
Based on a PR #79.

We only need the following files:
- consent.proto
- deal.proto
- coin.proto
- gogo.proto

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -14,7 +14,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.0
 	github.com/ipfs/go-ipfs-api v0.3.0
 	github.com/lestrrat-go/jwx/v2 v2.0.8
-	github.com/medibloc/panacea-core/v2 v2.0.6-0.20230113040800-7e7a4728ab82
+	github.com/medibloc/panacea-core/v2 v2.0.6-0.20230202103417-de758e7f84e9
```
Comment (youngjoon-lee) [inline go.mod:17]: This must be updated once https://github.com/medibloc/panacea-core/pull/627 is merged.

File: third_party/proto/panacea/datadeal/v2/deal.proto [hunk 2]
```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 package panacea.datadeal.v2;
 
-option go_package = "github.com/medibloc/panacea-core/x/datadeal/types";
+option go_package = "github.com/medibloc/panacea-core/v2/x/datadeal/types";
```
Comment (youngjoon-lee) [inline third_party/proto/panacea/datadeal/v2/deal.proto:4]: I didn't update this manually. I just copied this file from https://github.com/medibloc/panacea-core/pull/627 :)

**Comment by youngjoon-lee** [comment]
I'm also trying to adopt `buf` for dependency management. Coming soon. Please focus on rate limiting while I am working on `buf`.

**Comment by youngjoon-lee** [comment]
> I'm also trying to adopt `buf` for dependency management. Coming soon. Please focus on rate limiting while I am working on `buf`.

@gyuguen I've tried to use `buf`: https://github.com/medibloc/panacea-oracle/commit/4cc967394481c2d1a2cc2261b1186fc376be4cf4, but there are some limitations for now because we're using cosmos-sdk < v0.46 that doesn't publish their proto files to [BSR](https://buf.build/cosmos/cosmos-sdk) (Buf Schema Registry). That is, we can use `buf` with a complete dependency management once we upgrade cosmos-sdk to v0.46+. Until then, let's use our copy&paste strategy.

**Comment by gyuguen** [comment]
Yes. it's okay it'll be fine
I've used 'buf' briefly before, and it's a handy feature.

And I will fix Lint at #79

File: go.mod (hunk 1 above)
Comment (gyuguen) [reply go.mod:17]: I got it. 
I'll change this version once the panacea core PR merge.

**Review by gyuguen — APPROVED** [review]
LGTM
Thank you!

File: go.mod (hunk 1 above)
Comment (youngjoon-lee) [reply go.mod]: Done 3a4af56ac1e11005315723984b2baf9acbdb3e8d

**Pull request merged**